# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.ini
# Commit: 8c70bc92ad36
# Full Hash: 8c70bc92ad36beb61f2cd6031e80ace3793fe53b
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 1 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r c6f1243c3ba4 -r 8c70bc92ad36 testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.ini	Wed Feb 23 20:49:26 2022 +0000
@@ -0,0 +1,3 @@
+[mix-blend-mode-both-parent-and-blended-with-3D-transform.html]
+  fuzzy:
+    maxDifference=94-95;totalPixels=340-460